pkg/components/prometheus-operator/component.go

```diff
@@ -57,7 +57,11 @@ type Grafana struct {
 
```

Comment:
The second commit (3c1689f5b921dd87b25b5e367178c8e1017fb258) says `Unverified`.

Reply:
This is fine. @knrt10 does not sign it's commits and my account requires a signature, as I always sign.

Reply:
Signed now. On next push it should be verified I guess